pkg/kubeinteraction/resultsannotation.go

```diff
@@ -0,0 +1,48 @@
+package kubeinteraction
```

Comment:
there is a e2e test to update as well to make sure the annotation was added and matches the sha properly